### ISSUE TITLE
Error on Gradle sync

### ISSUE THREAD

**Issue by dogan-gulcan-zocdoc** [browserstack/browserstack-gradle-plugin#18]
After following installation steps and adding the following to my modules `build.gradle` file;
```
buildscript {
    repositories {
        ...
        maven { url "https://plugins.gradle.org/m2/" } // for com.browserstack.gradle:browserstack-gradle-plugin
    }

    dependencies {
        ...
        classpath "gradle.plugin.com.browserstack.gradle:browserstack-gradle-plugin:2.3.1"
    }
}
...
apply plugin: "com.browserstack.gradle"
...

browserStackConfig {
    username = "..."
    accessKey = "..."
    devices = ["Google Pixel 3-9.0", "Samsung Galaxy S9 Plus-8.0", "Google Nexus 6-6.0"]
}
```

I'm having the following error on my Gradle sync.
```
WARNING: Task 'runDebugBuildOnBrowserstack' is deprecated now. Please use new task 'executeDebugTestsOnBrowserStack' instead of this. Please refer https://github.com/browserstack/browserstack-gradle-plugin/blob/master/README.md for more details.
WARNING: Task 'uploadBuildToBrowserstackAppLive' is deprecated now. Please use new task 'uploadDebugToBrowserstackAppLive' instead of this. Please refer https://github.com/browserstack/browserstack-gradle-plugin/blob/master/README.md for more details.
ERROR: null
Daemon vm is shutting down... The daemon has exited normally or was terminated in response to a user interrupt.
----- End of the daemon log -----


FAILURE: Build failed with an exception.
```

Is there anything amiss with the implementation? How can we solve this? Thanks!

Edit: if I use version `2.3.0` instead, Gradle syncs properly.

**Comment by cptbucky** [comment]
It's due to a null pointer exception on this line: 

https://github.com/browserstack/browserstack-gradle-plugin/blob/master/src/main/java/com/browserstack/gradle/BrowserStackConfigExtension.java#L114

The `appStoreConfiguration` field in BrowserStackConfigExtension.java is not initialised however the above check is expecting it to be set and also populated with username and password. 

It looks like an optional behaviour from the docs: https://www.browserstack.com/app-automate/capabilities therefore I assume that the field itself should be initialised as empty and so the following: 

https://github.com/browserstack/browserstack-gradle-plugin/blob/master/src/main/java/com/browserstack/gradle/EspressoTask.java#L249 

would then pass `[]` to the api, happy to contribute if I'm correct in my approach.

**Comment by ralfgehrer** [comment]
Having the same issue also on 2.3.0. Any fix coming up ?

**Comment by raghuhit** [comment]
Sorry for delayed response, have fixed this issue in v2.3.2 of the plugin. Closing this task.
